public/app/features/correlations/types.ts

```diff
@@ -26,18 +26,41 @@ export interface RemoveCorrelationResponse {
   message: string;
 }
 
-export type CorrelationType = 'query';
+type CorrelationTypeOptions = {
+  value: string;
+  label: string;
+  description: string;
+};
+
+export const CORR_TYPES: Record<string, CorrelationTypeOptions> = {
+  query: {
+    value: 'query',
+    label: 'Query',
+    description: 'Open a query',
+  },
+  external: {
+    value: 'external',
+    label: 'External',
+    description: 'Open an external URL',
+  },
+};
+
+const corrTypeArray = Object.values(CORR_TYPES).map((ct) => ct.value);
+
+export type CorrelationType = (typeof corrTypeArray)[number];
```

Comment:
This type chain also seems a bit weird to me. It seems like the idea was to create `type CorrelationType = 'query' | 'external'` but as far as I can tell the `CorrelationType` is just `string` so it's not really helpful.